Shopify/dawn
# [Animations] Fix deferred media button hovers when 3D Lift is active

### PR Summary: 

Fix the display of button hovers on deferred media when 3D Lift is active. 

### Why are these changes introduced?

Currently, when the "3D Lift" hover effect is active, deferred media buttons on products slide down and to the left on hover. It can also get kinda glitchy depending on where you hover. This is unintentional. 😅 (Thanks @melissaperreault for [reporting this](https://github.com/Shopify/dawn/pull/2575#pullrequestreview-1402620597)!)

### What approach did you take?

I considered just removing the hover state from those buttons entirely, but it was easy enough to target a more universal class for deferred media. It feels more cohesive to have the hover effect active on the play buttons. 

### Visual impact on existing themes

Before


https://user-images.githubusercontent.com/1202812/234917700-aa488d5c-c5cb-4360-92f0-2622754d405a.mp4



After


https://user-images.githubusercontent.com/1202812/234917331-e8699242-910e-417f-8bdd-8aab3d92b770.mp4



### Testing steps/scenarios

- Activate the "3d Lift" hover effect from within Theme Settings > Animations
- Visit a product that uses a video or 3D model. Ensure that the play button animates as intended. 
- Hover over the play button for a Video section. Ensure that also animates as intended. 

### Demo links

- [Store](https://os2-demo.myshopify.com/?preview_theme_id=139854970902)
- [Editor](https://os2-demo.myshopify.com/admin/themes/139854970902/editor)

### Checklist
